libxenon/drivers/xenon_nand/xenon_sfcx.c

```diff
@@ -202,7 +202,7 @@ void sfcx_calcecc_ex(unsigned int *data, unsigned char* edc) {
 	val = ~val;
 
 	// 26 bit ecc data
-	edc[0] = ((val << 6) | (data[0x20C] & 0x3F)) & 0xFF;
+	edc[0] = ((val << 6) | (edc[0] & 0x3F)) & 0xFF;
```

Comment:
```suggestion
	edc[0] = (val << 6) & 0xFF;
```

According to the C# code i have that does the equivalent, this is the correct way to do this, if someone with a hardware flasher easily hooked up could test that, that'd be great.

Reply:
With this; https://github.com/Swizzy/x360Utils/blob/master/src/x360Utils/NAND/NANDSpare.cs#L123 doing the same calculation i'm feeling more confident that this is correct, i've verified this using a recent nand dump of my Winchester.

Reply:
@Swizzy verified it myself, both rawflash and updxell behave as expected with this change and i'm not getting any bad block errors when reading back the NAND on my falcon JTAG. As such i've committed the suggestion to the PR branch.